beam/src/main/java/org/apache/zeppelin/beam/StaticRepl.java

```diff
@@ -147,27 +147,10 @@ public static String execute(String generatedClassName, String code) throws Exce
 
         return baosOut.toString();
 
-      } catch (ClassNotFoundException e) {
-        logger.error("Exception in Interpreter while Class not found", e);
-        System.err.println("Class not found: " + e);
-        e.printStackTrace(newErr);
-        throw new Exception(baosErr.toString());
-
-      } catch (NoSuchMethodException e) {
-        logger.error("Exception in Interpreter while No such method", e);
-        System.err.println("No such method: " + e);
-        e.printStackTrace(newErr);
-        throw new Exception(baosErr.toString());
-
-      } catch (IllegalAccessException e) {
-        logger.error("Exception in Interpreter while Illegal access", e);
-        System.err.println("Illegal access: " + e);
-        e.printStackTrace(newErr);
-        throw new Exception(baosErr.toString());
-
-      } catch (InvocationTargetException e) {
-        logger.error("Exception in Interpreter while Invocation target", e);
-        System.err.println("Invocation target: " + e);
+      } catch (ClassNotFoundException | NoSuchMethodException | IllegalAccessException
+          | InvocationTargetException e) {
+        logger.error("Exception in Interpreter while execution", e);
+        System.err.println(e);
         e.printStackTrace(newErr);
         throw new Exception(baosErr.toString());
```

Comment:
could/should this Exception include the original?
https://docs.oracle.com/javase/7/docs/api/java/lang/Exception.html#Exception(java.lang.String,%20java.lang.Throwable)